src-ghc/Pact/Server/PactService.hs

```diff
@@ -64,36 +65,52 @@ initPactService CommandConfig {..} loggers = do
 
 applyCmd :: Logger -> Maybe EntityName -> PactDbEnv p ->
             GasModel -> Word64 -> Int64 -> ExecutionMode -> Command a ->
-            ProcessedCommand PublicMeta ParsedCode -> IO CommandResult
-applyCmd _ _ _ _ _ _ _ cmd (ProcFail s) = return $ jsonResult Nothing (cmdToRequestKey cmd) (Gas 0) s
+            ProcessedCommand PublicMeta ParsedCode -> IO (CommandResult [TxLog Value])
+applyCmd _ _ _ _ _ _ _ cmd (ProcFail s) =
+  return $ resultFailure Nothing (cmdToRequestKey cmd) (Gas 0) $
+  PactError TxFailure def def . viaShow $ s
```

Comment:
The identation here may work, but it's a little confusing because it's the same column as the actual return statement. Do you mind bumping `PactError...` over a tab for legibility?

Reply:
See latest commit. Also added some other edits for better legibility.